Update readme.md (#1825)
kamranahmedse/developer-roadmap: content/roadmaps/106-javascript/content/107-javascript-control-flow/101-conditional-statements/readme.md

> # Conditional statements 
> 
> Very often when you write code, you want to perform different actions for different decisions. You can use conditional statements in your code to do this. In JavaScript we have three conditional statements:
> 
>   `if` statement - use this statement if you want to execute a set of code when a condition is true
>   
>   `if...else` statement - use this statement if you want to select one of two sets of lines to execute
>   
>   `switch` statement - use this statement if you want to select one of many sets of lines to execute
> 
> 
> <ResourceGroupTitle>Free Content</ResourceGroupTitle>
> <BadgeLink colorScheme='yellow' badgeText='Read' href='http://www.cev.washington.edu/lc/CLWEBCLB/jst/js_conditionals.html)](http://www.cev.washington.edu/lc/CLWEBCLB/jst/js_conditionals.html'>JavaScript Conditional Statements</BadgeLink>
> <BadgeLink colorScheme='yellow' badgeText='Read' href='https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/conditionals'>Making decisions in your code — conditionals</BadgeLink>

Reviewer: ```suggestion
# Conditional statements 

When you write code, you often want to perform different actions for different decisions. You can use conditional statements in your code to do this. In JavaScript, we have three conditional statements: `if`, `if...else`, and `switch`.

<ResourceGroupTitle>Free Content</ResourceGroupTitle>
<BadgeLink colorScheme='yellow' badgeText='Read' href='https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/conditionals'>Making decisions in your code — conditionals</BadgeLink>
<BadgeLink colorScheme='yellow' badgeText='Read' href='https://javascript.info/ifelse'>Conditional branching: if, '?'</BadgeLink>
```
